Source/Dafny/Verifier/Translator.cs

```diff
@@ -8260,12 +8269,12 @@ public ForceCheckToken(IToken tok)
         tok = ForceCheckToken.Unwrap(tok);
         var args = new List<object>();
         args.Add(Bpl.Expr.Literal(0));
-        Bpl.AssertCmd cmd = TrAssertCmd(tok, condition, new Bpl.QKeyValue(tok, "subsumption", args, kv));
-        cmd.ErrorData = "Error: " + errorMessage;
+        Bpl.AssertCmd cmd = TrAssertCmdDesc(tok, condition, desc, new Bpl.QKeyValue(tok, "subsumption", args, kv));
         return cmd;
       }
     }
 
+    // TODO: update to include structured description
```

Comment:
Ah, no, it's just a further progression of this refactoring that remains to be done. I'm thinking of doing it in a future PR, since this one is functional, to break things down into more bite-sized pieces.